data/datasets/README.md

```diff
@@ -48,6 +48,18 @@ proposed. The text-only dataset must include the following columns:
 2. **SOURCE** (string)
 3. **METADATA** (JSON string, optional)
 
+### **Prosocial format**
+
+For datasets that are intended to be used to train safety models, prosocial
+format is proposed. The format is given below
+
+1. **USER** (string)
+2. **RESPONSE** (string, optional)
+3. **ROTs** (List)
+4. **SAFETY_LABEL** (string - categorical)
+5. **EPISODE_DONE** (bool)
+6. **SOURCE** (string,optional)
```

Comment:
This needs more detailed description. I didn't realize it until I checked ProsocialDialog paper.

Reply:
@Vechtomov  Done.

Reply:
Can you also add a link to the paper?